cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -73,11 +73,13 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
         ${ssh} node02 -- ip l show eth1
         ${ssh} node02 -- ip l show eth2
 
-        # Verify Multus v3 image is used
-        ${ksh} get ds -n kube-system kube-multus-ds -o yaml | grep multus-cni:v3
+        # Verify Multus v4 image is used
+        ${ksh} get ds -n kube-system kube-multus-ds -o yaml | grep multus-cni:v4
 
         # Sanity check that Multus is able to connect secondary networks
         ${ksh} create -f "$DIR/test-multi-net.yaml"
+        sleep 30
+        ${ksh} get pod test-multi-net -o yaml
         ${ksh} wait pod test-multi-net --for condition=ready=true
```

Comment:
nit: FYI wait accepts --timeout flag, default is 30s
Multus is image is 410MBs, 30s is the borderline when CI bandwith is not loaded, I would set at least 2m timeout.